eb

```diff
@@ -39,48 +39,35 @@
 # @author: Pieter De Baets (Ghent University)
 # @author: Jens Timmerman (Ghent University)
 
+PYTHON=python
+
 # Python 2.6 or more recent 2.x required
 REQ_MAJ_PYVER=2
 REQ_MIN_PYVER=6
 REQ_PYVER=${REQ_MAJ_PYVER}.${REQ_MIN_PYVER}
 
 # make sure Python version being used is compatible
-pyver=`python -V 2>&1 | cut -f2 -d' '`
+pyver=`$PYTHON -V 2>&1 | cut -f2 -d' '`
 pyver_maj=`echo $pyver | cut -f1 -d'.'`
-pyver_min=`echo $pyver | cut -f2 -d'.'`
+
+if [ $pyver_maj -ne $REQ_MAJ_PYVER ]
```

Comment:
yes, and it seems to me like the proper thing to do. We want python 2.x.  We shouldn't directly call python and assume it is 2.x. And if you have to different pythons (python and python2), then you're in a world of shit. We cannot account for every oddball case in the world? Most (if not all) distros  have a python2 and a python3 these days.